TestSolutions/Violate-Analyzers.ps1

```diff
@@ -1,6 +1,6 @@
 function Violate-Analyzers()
 {
-    "This file is intended to verify that PSScriptAnalyzer works and contains intentionally bad code."
+    Write-Host "This file is intended to verify that PSScriptAnalyzer works and contains intentionally bad code."
```

Comment:
```suggestion
    Write-Host "This file contains intentionally bad code to verify that PSScriptAnalyzer works correctly."
```

Reply:
LOL. I had the exact same change in stash from an earlier issue that I forgot to pop it and commit!